### PR TITLE
open-orders-mobile-updates

### PR DESCRIPTION
**Pull request by JohnDanz**
centered the quantity column for open orders on mobile, centered quantity values
https://github.com/AugurProject/augur/issues/5050

### CODE REVIEW

File: packages/augur-ui/src/modules/market/components/market-orders-positions-table/open-orders-table.styles.less [hunk 1]
```diff
@@ -2,6 +2,18 @@
 
 .Tabs {
   cursor: default;
+
+  @media @breakpoint-mobile {
+    > div[class*="Content"] > div.Table {
+      > ul[class*="DataTableHeader"] > li:nth-of-type(3) {
```
Comment (phoebemirman) [inline packages/augur-ui/src/modules/market/components/market-orders-positions-table/open-orders-table.styles.less]: Can you just add the style in DataTableHeader.styles.less and Row.styles.less?

Open orders is .Row for that one. 

Shouldn't be this complicated

**Review by phoebemirman — CHANGES_REQUESTED** [review]

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://dashboard.featurepeek.com). 🎉
Your deployment will be kept up-to-date with this pull request's latest changes.

**https://peek.run/1reh3np**

-------

*Please read [our docs](https://docs.featurepeek.com/configuring-peek-yml/) for more configuration details.*

File: packages/augur-ui/src/modules/market/components/market-orders-positions-table/open-orders-table.styles.less (hunk 1 above)
Comment (JohnDanz) [reply packages/augur-ui/src/modules/market/components/market-orders-positions-table/open-orders-table.styles.less]: agreed, will update

File: packages/augur-ui/src/modules/market/components/market-orders-positions-table/open-orders-table.styles.less [hunk 2]
```diff
@@ -23,6 +23,12 @@ div.Table {
       min-height: unset;
     }
   }
+
+  @media @breakpoint-mobile {
+    > ul[class*="DataTableHeader"] > li:nth-of-type(3) {
```
Comment (phoebemirman) [inline packages/augur-ui/src/modules/market/components/market-orders-positions-table/open-orders-table.styles.less]: can you put this in DataTableHeader styles file?

**Review by phoebemirman — APPROVED** [review]

**Pull request merged**